File: BackUpForABook/UIControls/Common/ANDataTableCell.m
//
//  ANDataTableCell.m
//  ANBookPad
//
//  Created by hui wang on 9/3/16.
//  Copyright © 2016 MH.dingf. All rights reserved.
//

#import "ANDataTableCell.h"

@implementation ANDataTableCell

// create one cell
+(ANDataTableCell*)createWith:(NSString*)unit
                       symbol:(NSString*)symbol
                        value:(double)value
                        image:(nullable UIImage*)image
                     viewSize:(int)viewSize
                          tip:(NSString*)tip
{
    ANDataTableCell* oneCell = [[ANDataTableCell alloc] init];
    
    oneCell.unit_name = unit;
    oneCell.unit_symbol = symbol;
    oneCell.value = value;
    oneCell.image = image;
    oneCell.viewSize = viewSize;
    oneCell.tip = tip;
    
    return oneCell;
}

@end

File: BackUpForABook/UIControls/TRClasses_swift/ANDataTableCell.swift
//
//  ANDataTableCell.swift
//  TreeRingMap_Swift
//
//  Created by dingf on 17/3/17.
//  Copyright © 2017年 MH.dingf. All rights reserved.
//

import UIKit

class ANDataTableCell: NSObject {
    var unit_name: String?
    var unit_symbol: String?
    var value: Double?
    var image: UIImage?
    var viewSize: Int?
    var tip: String?
    
    class func create(_ unit: String, symbol: String, value: Double, image: UIImage?, viewSize: Int, tip: String) -> ANDataTableCell{
        let oneCell = ANDataTableCell()
        oneCell.unit_name = unit
        oneCell.unit_symbol = symbol
        oneCell.value = value
        oneCell.image = image
        oneCell.viewSize = viewSize
        oneCell.tip = tip
        return oneCell
    }

}
